b2sdk/session.py

```diff
@@ -41,6 +41,7 @@ class B2Session(object):
         to methods of underlying raw_api and reauthorizes if necessary.
     """
     SQLITE_ACCOUNT_INFO_CLASS = staticmethod(SqliteAccountInfo)
+    B2HTTP_CLASS = staticmethod(B2Http)
```

Comment:
shouldn't this be in ApiConfig? We'll have a switchable b2http "soon" with a curl backend, it has to be runtime-settable. If you will now make it injectable like in this PR, that will be harder to change later.

Reply:
That's a good point. My idea was to make the http backend pluggable by providing the `http_session_factory` a parameter-less callable that return a requests.Session-compliant object. It seems that having both `B2Http` and `http_session_factory` pluggable in the config is redundant. What do you think?